docs/OpenBSD.md

```diff
@@ -44,27 +52,27 @@ Install OpenBSD over PXE
 
   ```
   label open-bsd
-  	menu label ^OpenBSD-5.9
+  	menu label ^OpenBSD-6.1
   	menu default
   	kernel OpenBSD/amd64/pxeboot.0
   ```
-  Path to `bsd.rm` is set in `/etc/boot.conf`, which needs to be created.
-
+  
+  Note that it has to be in tftpd root directory, not the directory where bsd
+  files are!
+  
+  If you don't want to use DHCP, you can use the following commands in iPXE:
   ```
-  boot tftp:/OpenBSD/amd64/bsd.rd
+  iPXE> ifopen net0
+  iPXE> set net0/ip 192.168.0.100
+  iPXE> set net0/netmask 255.255.255.0
+  iPXE> set net0/gateway 192.168.0.1  
+  iPXE> chain tftp://192.168.0.1/auto_install
   ```
-  Note that it has to be in  tftpd root directory, not the directory where bsd
-  files are!
-
+  You need to adjust IP's and netmask to your own environment.
 3. Boot configuration.
```

Comment:
@pkubaj add a new line before "3. Boot configuration" because it appends to the previous line.